ee-feature-pack/galleon-shared/src/main/resources/feature_groups/domain-load-balancer.xml

```diff
@@ -15,15 +15,15 @@
     </feature>
 
     <feature spec="profile">
-        <param name="__profile" value="load-balancer"/>
+        <param name="profile" value="load-balancer"/>
             <feature-group name="logging">
                 <include feature-id="subsystem.logging.root-logger.ROOT:subsystem=logging,root-logger=ROOT">
                     <param name="handlers" value="[FILE]"/>
                 </include>
                 <exclude spec="subsystem.logging.console-handler"/>
             </feature-group>
             <feature-group name="io"/>
-            <feature-group name="undertow-load-balancer"/>
+            <feature-group name="undertow-load-balancer"/>        
```

Comment:
if you ever push to this branch again - don't reintroduce this formatting error

Reply:
Ups, I picked up the same hash at [ac3d819](https://github.com/wildfly/wildfly/pull/19468/commits/ac3d819954ef65ec957dbcfe56158033bfe96cf6) . Since it is a revert, the fix was reverted.

Since this is blocking WildFly Core testsuite, I propose to merge this now and fix the formatting error after.